libs/s25main/ogl/glTexturePacker.cpp

```diff
@@ -97,7 +97,7 @@ bool glTexturePacker::packHelper(std::vector<glSmartBitmap*>& list)
             }
             // free texture packer, as it is not needed any more
             root->destroy(list.size());
```

Comment:
is it possible to wrap that call into the node destructor itself?

Reply:
No. The destroy function does a depth-first loop to destruct the tree. The dtor would do a breath-first recursive call likely killing the stack memory.

I was thinking about improving this but due to the tree-structure this packer is very slow and unusable in debug builds of MSVC (takes minutes to build the texture). Hence I'm planning to rewrite it using a flat allocation and less smart distribution strategy but which seems to be still very good

Reply:
ok :-)